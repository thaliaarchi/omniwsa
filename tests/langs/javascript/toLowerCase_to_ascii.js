console.log("Characters which String.prototype.toLowerCase maps to ASCII:");
for (let code = 0x80; code <= 0x10FFFF; code++) {
  const ch = String.fromCodePoint(code);
  const lower = [...ch.toLowerCase()];
  if (lower.every(ch => ch.codePointAt(0) <= 0x7f)) {
    console.log(`${ch} -> ${lower}`);
  }
}
