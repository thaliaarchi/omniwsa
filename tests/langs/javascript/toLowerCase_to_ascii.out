Node.js v23.3.0
Characters which String.prototype.toLowerCase maps to ASCII:
â„ª -> k
